packages/domain/lib/base/mappers/base_mappers/mapper_base_two_in.dart

```diff
@@ -0,0 +1,3 @@
+abstract class MapperTwoIn<I, ITwo, O> {
+  O call(I params, ITwo twoParams);
+}
```

Comment:
nope. keep the same approach as in usecase. if u need to pass several params, wrap them with the new model, like:

class C {
  final A a;
  final B b;
}